pkg/prompts/prompts.go

```diff
@@ -46,6 +80,17 @@ func validateExistingFilepath(input string) error {
 	return errors.New("file doesn't exist")
 }
 
+func validateWeight(input string) error {
+	val, err := strconv.ParseUint(input, 10, 64)
+	if err != nil {
+		return err
+	}
+	if val < 1 || val > 100 {
```

Comment:
is this an avalanchego constraint?

Reply:
Yes: https://docs.avax.network/apis/avalanchego/apis/p-chain#platformaddsubnetvalidator, there is a `weight` section after the example